integration-test/src/test/kotlin/io/cloudstate/kotlinsupport/tests/IntegrationTest.kt

```diff
@@ -6,17 +6,17 @@ import akka.stream.ActorMaterializer
 import akka.stream.Materializer
 import com.example.shoppingcart.ShoppingCartClient
 import com.example.shoppingcart.ShoppingCartProto
-import io.cloudstate.kotlinsupport.logger
 import org.junit.Rule
 import org.junit.Test
+import org.slf4j.LoggerFactory.getLogger
 import org.testcontainers.containers.FixedHostPortGenericContainer
 import org.testcontainers.containers.output.Slf4jLogConsumer
 import org.testcontainers.containers.wait.strategy.Wait
 import java.util.concurrent.TimeUnit
 import kotlin.test.assertEquals
 
 class IntegrationTest {
-    private val log = logger()
```

Comment:
If we could find a way to take advantage of ActorLogging from the java support library we could remove that from all the code and slf4j dependencies of project. If you want to include an issue to investigate this possibility feel free :D